p2p/module.go

```diff
@@ -127,6 +169,26 @@ func (m *p2pModule) Start() error {
 	return nil
 }
 
+func (m *p2pModule) getAddrBookPerHeight(height uint64) (typesP2P.AddrBook, error) {
+	persistenceReadContext, err := m.bus.GetPersistenceModule().NewReadContext(int64(height))
+	if err != nil {
+		return nil, err
+	}
+	vals, err := persistenceReadContext.GetAllValidators(int64(height))
```

Comment:
Either add a TODO to get other staked actors too (fisherman, service nodes, etc...) or incorporate it here.

Reply:
Maybe addressed in https://github.com/pokt-network/pocket/pull/374/commits/36d7129cfc1c0199ddb1931438c16b6480dda88e